src/Contexts/view.reducer.jsx

```diff
@@ -60,7 +61,7 @@ export function viewReducer(state = viewInitialState, { type, payload }) {
             }
 
 
-        case "BOOTSTRAP_FINISH":
+        case BOOTSTRAP_FINISHED:
```

Comment:
Eine Dokumentation um was sich die einzelnen Reducer kümmern sollen, wäre bei allen context reducern auch noch interessant. Irgendwo gleich oben in den Files. Damit legen wir uns selbst die Schranken fest und machen den Code gleichzeitig leichter verständlich. 
Können wir aber beide zusammen dann ausformulieren.

Reply:
Aufhänger war das ich den view immer mit dem viewport vom backend gleich setzte, aber er hier eine ganz andere Verwendung hat.

Reply:
hab was hinzugefügt, bin mir nicht sicher ob es dir taugt, bin gerade nur etwas einfallslos